samples/matter/light_bulb/src/app_task.cpp

```diff
@@ -103,7 +103,15 @@ CHIP_ERROR AppTask::Init()
 		return err;
 	}
 
+#ifdef CONFIG_OPENTHREAD_MTD
+#ifdef CONFIG_OPENTHREAD_MTD_SED
```

Comment:
`OPENTHREAD_MTD_SED` depends on `OPENTHREAD_MTD` so I think we can reduce nesting of `#ifdefs` here.

Reply:
Yes, I know it does, but how would we separate sleepy MTD from non-sleepy MTD in that case?

Reply:
Ok, light bulb doesn't even support SED configuration, so i will just remove it

Reply:
I meant sth like:
```
#ifdef CONFIG_OPENTHREAD_MTD_SED

#elif CONFIG_OPENTHREAD_MTD

#else
```